packages/clay-css/src/scss/_bs4.scss

```diff
@@ -17,7 +17,7 @@
 @import 'bootstrap/nav';
 @import 'bootstrap/navbar';
 @import 'bootstrap/card';
-@import 'bootstrap/breadcrumb';
+// @import 'bootstrap/breadcrumb';
```

Comment:
Out of curiosity, why are we commenting these as opposed to deleting the lines?

Reply:
@wincent That is a good question. It was to help me keep track of what components have been converted. Looking at it now, deleting the lines would accomplish the same haha. At the end of the conversion this, this file is going away.

Reply:
![kill-it-with-fire](https://user-images.githubusercontent.com/7074/86141958-f2686000-baf2-11ea-9402-49e234564737.gif)